packages/app-desktop/gui/styles/inline-markdown.scss

```
@@ -0,0 +1,37 @@
+	position: relative;
+
+	// Ensure images don't overflow the container
+	img {
+		max-width: 100%;
+	}
+
+	// Decrease space between paragraphs to better fit
+	p {
+		margin-bottom: var(--joplin-margin);
+	}
+	// Avoid a large space at the end of the chat bubble after
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)